src/hotspot/cpu/ppc/macroAssembler_ppc.cpp

```diff
@@ -24,6 +24,7 @@
  */
 
 #include "precompiled.hpp"
+#include "asm/assembler.inline.hpp"
```

Comment:
I think this is not needed because it is included via `macroAssembler.inline.hpp`.